### DISCUSSION TITLE
Error response from daemon:  read-only file system

### DISCUSSION THREAD

**Discussion by Prince-1501** [abiosoft/colima#289] [Q&A] [answered]
How i can resolve this issue can anyone help me with this?

```
Error response from daemon: error while creating mount source path '/Users/princekumar/Desktop/browserstack/hawkeye/logs': mkdir /Users/princekumar/Desktop/browserstack/hawkeye/logs: read-only file system

```
Let's suppose if i run ``` mkdir /Users/princekumar/Desktop/browserstack/hawkeye/logs ```
then it's shows the next error like this 

Error response from daemon: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:76: mounting "/Users/princekumar/Desktop/browserstack/hawkeye/frontend" to rootfs at "/usr/src/service/frontend" caused: mkdir /var/lib/docker/overlay2/1a909619ccf7fd317689e8af5adeb94077e5d6f87cc6e7748d43ac5040aaf9dc/merged/usr/src/service/frontend: read-only file system: unknown

My system config

limactl version 0.10.0
colima version v0.3.4
Macos Monterey v12.2 ( M1 Chip )

**Comment by abiosoft** [comment, marked as answer]
How did you start colima? kindly share the full command.

Also, can you update to v0.4.1 and see if the error remains?
```sh
colima delete # delete existing instance
brew upgrade colima
colima start
```

**Reply by Prince-1501** [reply]
Thanks @abiosoft  for reply back
I started Colima in this way 

```
exec arch -arm64e /bin/zsh
```

```
eval "$(/opt/homebrew/bin/brew shellenv)"
```

```
limactl delete default; limactl start
```

```
colima start --arch aarch64 --dns=8.8.8.8 --mount <Directory where repo is located>
```

**Reply by Prince-1501** [reply]
and upgrading Colima 0.4.1 is also giving the same error

**Reply by abiosoft** [reply]
You do not need the mount flag, the default suffices as it mounts your $HOME directory.
If you must use it, suffix it with `:w` to make it writable e.g. `$HOME/projects:w`

**Reply by Prince-1501** [reply]
Thanks @abiosoft it works for me 
when I start Colima with this `colima start --arch aarch64 --dns=1.1.1.1`
